src/components/input/MetadataInput.tsx

```diff
@@ -23,16 +27,33 @@ export const MetadataKeys = ServiceMetadata.type.types.reduce(
   [] as readonly string[]
 );
 
+const InputTextMetadataKeys = MetadataKeys.filter(
+  k => k !== "scope" && k !== "description"
+);
+
 const MetadataInput = ({
   service_metadata,
   onChange,
   isApiAdmin,
   t
 }: Props) => {
   return isApiAdmin ? (
-    // All input text Metadata (except 'scope' that is an enumeration)
+    /* - Input text: all metadata except 'scope' and 'descrition'
+     * - Text area: 'descrition' according to MetadataDescritionEditor
+     * - Select: 'scope' that is an enumeration
+     */
```

Comment:
Along with this comment, as the UI logic is getting articulated, we can explicit the behaviour by writing the very same, but as code:
```ts
MetadataKeys.map((k, i) => (
   k === "scope" ? <SelectInput  ...  />
   k === "description" ? <TextAreaInput  ...  />
   : <TextInput ... />
);
```
(I'm assuming order is not important. Otherwise, sort `MetadataKeys`).